composer.json

```diff
@@ -17,8 +17,8 @@
         "symfony/console": "~2.1|~3.0",
         "symfony/form": "~2.1|~3.0",
         "symfony/property-access": "~2.3|~3.0",
-        "ruflin/elastica": "~2.1",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "ruflin/elastica": "3.1.*"
```

Comment:
Thanks solivier, I've changed that line. I've leaved reference to php 5.4 to maintain comply with elastica.